test/TestServerProjects/media_types/Generated/MediaTypesClient.cs

```diff
@@ -265,7 +265,7 @@ public virtual Response<string> BinaryBodyWithTwoContentTypes(ContentType1 conte
         /// <param name="contentType"> Upload file type. </param>
         /// <param name="message"> The payload body. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<string>> BinaryBodyWithThreeContentTypesAsync(ContentType1 contentType, Stream message, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<string>> BinaryBodyWithThreeContentTypesAsync(ContentType1AutoGenerated contentType, Stream message, CancellationToken cancellationToken = default)
```

Comment:
This comes from M4 change. 
![image](https://user-images.githubusercontent.com/97225342/162390777-b65b2299-b06b-43e5-8b36-9f28f3ba4bd4.png)
@timotheeguerin do you have any ideas of this change.

Reply:
Could we preview the changes this generator update will have in the azure-sdk-for-net repo before we merge this change?  I'm interested to know whether it will make any unexpected API changes there.

Reply:
@annelo-msft I put at PR https://github.com/Azure/azure-sdk-for-net/pull/28363. I reviewed all the changes and all look good to me. I have comments to explain the changes in PR. Please check out.